tests/func/test_import_url.py

```diff
@@ -267,3 +267,26 @@ def test_import_url_to_remote_status(tmp_dir, dvc, local_cloud, local_remote):
 
     status = dvc.status()
     assert len(status) == 0
+
+
+def test_import_url_no_download(tmp_dir, dvc, workspace):
+    workspace.gen("file", "file content")
+    dst = tmp_dir / "file"
+    stage = dvc.imp_url(
+        "remote://workspace/file", os.fspath(dst), no_download=True
+    )
+
+    assert not dst.exists()
+    assert stage.deps[0].get_hash().value == "d10b4c3ff123b26dc068d43a8bef2d23"
+
+
+def test_import_url_no_download_status(tmp_dir, dvc, workspace):
+    workspace.gen("file", "file content")
+    dst = tmp_dir / "file"
+    dvc.imp_url("remote://workspace/file", os.fspath(dst), no_download=True)
+
+    status = dvc.status()
+    assert status["file.dvc"] == [
+        {"changed outs": {"file": "deleted"}},
+        "changed checksum",
+    ]
```

Comment:
Probably can just move this into the test above. using `status` is just an additional check to the previous test, not really a new test.